app/models/person.rb

```diff
@@ -424,6 +424,18 @@ def has_active_enrollment
     false
   end
 
+  def ethnicity
+    # Retrieve the raw 'ethnicity' value from the database
+    # Return an empty array if 'ethnicity' is nil, ensuring consistent array behavior
+    read_attribute(:ethnicity) || []
+  end
+
+  def tribe_codes
+    # Retrieve the raw 'tribe_codes' value from the database
+    # Return an empty array if 'tribe_codes' is nil, ensuring consistent array behavior
+    read_attribute(:tribe_codes) || []
+  end
+
```

Comment:
@saipraveen18 Don't we want to add defaults for the fields `:ethnicity` and `:tribe_codes`?